src/components/views/rooms/wysiwyg_composer/components/PlainTextComposer.tsx

```diff
@@ -52,6 +53,19 @@ export function PlainTextComposer({
     rightComponent,
     eventRelation,
 }: PlainTextComposerProps): JSX.Element {
+    const [isAutoReplaceEmojiEnabled, setIsAutoReplaceEmojiEnabled] = useState(
+        SettingsStore.getValue<boolean>("MessageComposerInput.autoReplaceEmoji"),
```

Comment:
I believe there's also `useSettingValue` to do the getting + watching + unsubscribing for you

Reply:
Ah I actually looked for something as such, obviously not hard enough. 😂 thanks Dave